docs/introduction/faq.md

```diff
@@ -0,0 +1,121 @@
+---
+id: faq-section
+title: FAQ
+---
+
+# Frequently Asked Questions
+
+## Table of Contents
+[What is the Talawa Project?](#q-what-is-the-talawa-project)  
+[Why should I use Talawa?](#q-why-should-i-use-talawa)  
+[What does 'Talawa' mean, how do I pronounce it?](#q-what-does-talawa-mean-how-do-i-pronounce-it)  
+[What services are provided by Talawa?](#q-what-services-are-provided-by-talawa)  
+[How is Talawa different from other community based organizations?](#q-how-is-talawa-different-from-other-community-based-organizations)  
+[Is Talawa safe?](#q-is-talawa-safe)  
+[What can a leader of a non-profit do with Talawa?](#q-what-can-a-leader-of-a-non-profit-do-with-talawa)  
+[What is Palisadoes?](#q-what-is-palisadoes)  
+[Where does Talawa store the Data?](#q-where-does-talawa-store-the-data)  
+[Is Talawa free?](#q-is-talawa-free)  
+[How do I use Talawa?](#q-how-do-i-use-talawa)  
+[How do I create account on Talawa?](#q-how-do-i-create-account-on-talawa)  
+[How do I list my organization on Talawa?](#q-how-do-i-list-my-organization-on-talawa)  
+[How can my organization benefit from Talawa?](#q-how-can-my-organization-benefit-from-talawa)
+
+---
+
+### What is the Talawa Project?
+Talawa project is open source software for managing Community Based Organizations
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### Why should I use Talawa?
+Talawa provides help to community based organizations collaborate with their membership. These organizations would include religious groups, non-profit charities, social groups and in limited cases, businesses.​
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### What does 'Talawa' mean, how do I pronounce it?
+Talawa is a Jamaican Patois word that means to be strong, fearless, strong-willed, and not to be underestimated. Jamaicans have always had a motto that encapsulates this idea of being great: 'We likkle but we tallawah' It means that we may be small, but we are strong, we are mighty, and can do anything. It is pronounced exactly as it is spelled: /Tala-wa/.
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### What services are provided by Talawa?
+Talawa has three main components:
+1. A mobile application with social media features
+2. A web based portal to be used by the organization’s administrative team, and
+3. An API providing access to data and features.​​
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### How is Talawa different from other community based organizations?
+Though software applications exist for these types of organizations (eg. Breeze ChMS, The Realm), they assume their memberships are fully literate and have access to email. But we wanted to create a system that would work for countries with similar technology and education challenges as Jamaica.​
```

Comment:
Edit this to: Many other similar softwares assume their memberships are fully literate and have access to email. We wanted to create a system that would work for countries with similar technology and education challenges as Jamaica.​ We also offer two levels of administrative roles, which allows more nuanced scheduling and planning for organizations with more than one chapter.